### PR TITLE
Ass-2_jsx/react_submission

### PR DESCRIPTION
**Pull request by DSdivyanksingh**
ass-2_react

### CODE REVIEW

**Comment by saikiransathpadi** [comment]
"correct, try to format the code to look more cleane
also don't give unnecessary new lines"

File: my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+        
+        
+
+
+    </div>
+  );
+}
+
+export default App;
```
Comment (imnashu) [inline my-app/src/App.js:41]: Looks perfect, All good!  
Thanks

Comment (DSdivyanksingh) [reply my-app/src/App.js:41]: thank you

**Comment by DSdivyanksingh** [comment]
> "correct, try to format the code to look more cleane also don't give unnecessary new lines"

sure ,sir & thank you

**Pull request merged**